code/controllers/master.dm

```diff
@@ -216,7 +216,7 @@ GLOBAL_VAR_INIT(CURRENT_TICKLIMIT, TICK_LIMIT_RUNNING)
 	//	local vars rock
 
 	// Schedule the first run of the Subsystems.
-	round_started = world.has_round_started()
+	local_round_started = SSticker.HasRoundStarted()
```

Comment:
keep the world.has_round_started() proc and have it forward to ssticker.

the point of it is to keep the mc modular enough somebody could rip it out and implement it in their project regardless of if the project is ss13